src/pybind/mgr/dashboard/controllers/home.py

```diff
@@ -87,7 +87,7 @@ def _parse_accept_language(self, accept_lang_header):
         result.sort(key=lambda l: l[0])
```

Comment:
Just out of curiosity: why does the rule not apply to this line and the following?

Reply:
not sure why. but it seems that flake8 is happy after this change..